### PR TITLE
Week 3 Step 4

### PR DESCRIPTION
**Pull request by ethanimooney**
TLDR; Removes old title-changing fetch, adds comment form and doPost() function to handle the submission of the form.
LINK: https://ethanimooney-step-2020.appspot.com/
To see the result once you submit the form go to /data

Comment.java is a comment class that I created and realized that I won't need until later on.

DataServlet.java now has a doPost() function that takes the input from the form (just the message for now) and inputs it into an ArrayList that gets passed through the doGet().

index.html - I formatted it so the changes are all just tabs, etc. I also added the FontAwesome license, and the comment form and comment display sections

script.js - no changes, just formatting

style.css - added styling for the comment sections

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -28,9 +27,32 @@
...
+    String message = getParameter(request, "cf-message", "");
+
+    dataList.add(message);
+    // Respond with the result.
+    response.setContentType("text/html;");
+    response.getWriter().println(message);
+    response.sendRedirect("/index.html");
+  }
+
+    /**
```
Comment (KSullivan42) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:44]: Comments should line up

Comment (ethanimooney) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Gotcha, fixed

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,77 +1,143 @@
...
+		<h1 class="header-item">Ethan Mooney</h1>
+		<div class="header-item" id="text-switcher" style="font-size: 30pt; color: #353535">your new best friend</div>
+	</div>
+
+	<div class="flex-container">
+		<div class="flex-item-mid" id="about">
+			<div class="about-container">
+				<div class="about-item-left">
+					<div class="about-item-left-item">
+						<img src="/images/ethan-p1.jpeg" alt="Ethan on bridge" width="100px">
```
[the remark this answers is not in the document]
Comment (shaquille-hall) [reply portfolio/src/main/webapp/index.html:66]: ref: https://google.github.io/styleguide/htmlcssguide.html#Indentation
